observability/observabilitytest/observabilitytest.go

```diff
@@ -43,27 +43,27 @@ func (cme *nopMetricsExporter) ExportView(vd *view.Data) {}
 
 // SetupRecordedMetricsTest does setup the testing environment to check the metrics recorded by receivers, producers or exporters.
 // The returned function should be deferred "defer SetupRecordedMetricsTest(t)()".
```

Comment:
nit: this comment should also be updated.

Reply:
Done.